molviewspec/molviewspec/nodes.py

```diff
@@ -438,6 +438,29 @@ class RepresentationParams(BaseModel):
     type: RepresentationTypeT = Field(description="Representation type, i.e. cartoon, ball-and-stick, etc.")
 
 
+class CartoonParams(RepresentationParams):
+    size_factor: Optional[float] = Field(description="Scales the corresponding visuals.")
+    tubular_helices: Optional[bool] = Field(description="Simplify corkscrew helices to tubes.")
+    # TODO support for variable size, e.g. based on b-factors?
```

Comment:
Adding `type: Literal["cartoon"] = "cartoon"` here prevent the issue you mentioned.

Reply:
Looked into this and updated `make_params` to raise an exception if there are "unconsumed" properties (https://github.com/molstar/mol-view-spec/pull/41/commits/a9b471b5909d6792e79be6b41357d20cd29e00e7). Doesn't affect other examples and, at least, gives users an error rather than silently ignoring potential mistakes.